polyA/_app.py

```diff
@@ -94,10 +95,23 @@ def run():
     # Load the substitution matrix
     # ----------------------------
 
+    with open(opts.alignments_file_path) as _infile:
+        with open(opts.alignments_file_path) as _infile:
```

Comment:
Why open it twice, and why shadow the `_infile` name?

Reply:
Whoops definitely didn't mean to open it twice